portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -33,30 +33,31 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** 
-* Servlet that deletes comment entities
-*
-* @param request which contains data used to identify a comment in order to
-* delete it
-* @param response
-*/
+ * Servlet that deletes comment entities
+ *
+ * @param request which contains data used to identify a comment in order to
+ * delete it
+ * @param response
+ */
 @WebServlet("/delete-comment")
 public class DeleteCommentServlet extends HttpServlet {
 
+  static final int MAX_NUM_BLOGS = 5;
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Receive input on which blog we are retrieving comments from
-    int maxNumBlogs = 5;
-    int blogNumber = getUserNum(request, "blog-number", 1, maxNumBlogs);
+    int blogNumber = getUserNum(request, "blog-number", 1, MAX_NUM_BLOGS);
     if (blogNumber == -1) {
       response.setContentType("text/html");
-      response.getWriter().println("Please enter an integer between " +  
-      1 + " to " + maxNumBlogs + ".");
+      response.getWriter().println("Please enter an integer between 1 to " + 
+          MAX_NUM_BLOGS + ".");
       return;
     }
 
     // Retrieve Comments from Datastore for the given blog post
     FilterPredicate filterBlogComments = new FilterPredicate("blogNumber", 
-    FilterOperator.EQUAL, blogNumber);
+      FilterOperator.EQUAL, blogNumber);
```

Comment:
A broken line should be double-indented; so +4 spaces in total. (one indent is 2 spaces, so 2 indents would be 4 spaces)